BGL/test/BGL/test_Has_member_clear.cpp

```diff
@@ -25,7 +25,7 @@ int main()
 
   CGAL_static_assertion(!Has_member_clear<with_clear_but_args>::value);
 
-  CGAL_static_assertion(!Has_member_clear<with_clear_but_const>::value);
```

Comment:
actually I don't understand that change

Reply:
The previous implementation of `Has_member_clean` was testing a specific signature: `void(C::*)(void)` (and not `void(C::*)(void) const`). The new implementation checks if the expression `c.clear()` is valid, and it can detect a non-const or a const member function. That is a difference in behavior. I hope that does not really matter.

... a method `void clear() const` would be strange, anyway: it is supposed to modify the object (by clearing it).

Reply:
Then, it does not really make sense to keep the line as this is not something that we want to enforce.

Reply:
As you want. In my opinion it does not really matter.